compliance_checker/imos/imos.py

```diff
@@ -738,7 +738,7 @@ def check_time_variable(self, dataset):
                                     dataset,
                                     IMOSCheck.CHECK_VARIABLE_ATTRIBUTE,
                                     result_name,
-                                    BaseCheck.HIGH)
+                                    BaseCheck.MEDIUM)
 
```

Comment:
@ggalibert While we're at it, we could add a more useful reasoning message, like "Time units don't match the IMOS recommended units 'days since 1950-01-01 00:00:00 UTC'".